src/components/projects/ManuscriptPageContainer.tsx

```diff
@@ -62,10 +65,19 @@ const ManuscriptPageContainer: React.FC = () => {
 
 const ManuscriptPageView: React.FC = () => {
   const can = usePermissions()
+  const [isTrackingOn, setTrackingOn] = useState(true)
+
+  const toggleTracking = () => {
+    setTrackingOn(!isTrackingOn)
```

Comment:
If I understand correctly, here we just toggle suggestions visibility and not really disable track changes. Taking in account that we actually have a case and logic where we indeed disable TC completely it may be a good idea to consider using a more specific naming here to avoid confusion (i mean in general in this function). Maybe something like tracking-visible/tracking-hidden.

Reply:
Makes sense. Updated!